src/Symfony/Bundle/FrameworkBundle/Resources/config/workflow.xml

```diff
@@ -27,5 +27,9 @@
         <service id="Symfony\Component\Workflow\Registry" alias="workflow.registry" />
 
         <service id="workflow.security.expression_language" class="Symfony\Component\Workflow\EventListener\ExpressionLanguage" />
+
+        <service id="Symfony\Bundle\FrameworkBundle\Command\WorkflowDumpCommand">
```

Comment:
moving this requires `$container->removeDefinition(WorkflowDumpCommand::class);` call to be updated, as console.xml is also conditionally loaded.

Reply:
Nice catch ;-)

Reply:
well it still needs to be removed if console is not available ;-) not sure about the current effect of this service without console available. I guess it doesnt hurt.

Reply:
Argh! I think we can keep command definition in `console.xml` and remove it if workflow is disabled.